test/functions/blockchainFunctionsTest.js

```diff
@@ -7,6 +7,7 @@ const ethAddress = '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2'
 const limit = 2
 const transactionType = 'ALL'
 const numberOfHolders = 10
+const metric = 'mvrv_usd_10y'
```

Comment:
needs to be in `getMetric_` describe block, it is not used outside of it

Reply:
Yes, you're right